amr-wind/actuator_line/NREL5MW_ALM_BD/setup/README.md

```diff
@@ -0,0 +1,277 @@
+True                   HubLoss     - Use the Prandtl hub-loss model? (flag) [unused when Wake_Mod=0 or 3]
+True                   TanInd      - Include tangential induction in BEMT calculations? (flag) [unused when Wake_Mod=0 or 3]
+False                  AIDrag      - Include the drag term in the axial-induction calculation? (flag) [unused when Wake_Mod=0 or 3]
+False                  TIDrag      - Include the drag term in the tangential-induction calculation? (flag) [unused when Wake_Mod=0,3 or TanInd=FALSE]
+"Default"              IndToler    - Convergence tolerance for BEMT nonlinear solve residual equation {or "default"} (-) [unused when Wake_Mod=0 or 3]
+100                    MaxIter     - Maximum number of iteration steps (-) [unused when Wake_Mod=0]
+```
+
+and the default Beddoes-Leishman parameters are used here:
+
+```
+======  Unsteady Airfoil Aerodynamics Options  ====================================================
+False                  AoA34       - Sample the angle of attack (AoA) at the 3/4 chord or the AC point {default=True} [always used]
+3                      UA_Mod      - Unsteady Aero Model Switch (switch) {0=Quasi-steady (no UA), 2=B-L Gonzalez, 3=B-L Minnema/Pierce, 4=B-L HGM 4-states, 5=B-L HGM+vortex 5 states, 6=Oye, 7=Boeing-Vertol}
+True                   FLookup     - Flag to indicate whether a lookup for f' will be calculated (TRUE) or whether best-fit exponential equations will be used (FALSE); if FALSE S1-S4 must be provided in airfoil input files (flag) [used only when UA_Mod>0]
+0                      UAStartRad  - Starting radius for dynamic stall (fraction of rotor radius [0.0,1.0]) [used only when UA_Mod>0; if line is missing UAStartRad=0]
+1                      UAEndRad    - Ending radius for dynamic stall (fraction of rotor radius [0.0,1.0]) [used only when UA_Mod>0; if line is missing UAEndRad=1]
+```
+
+**Beamdyn changes**
+
+In addition to the changes listed above, we also remove one of the blade stations from the beamdyn blade definition files.  The second blade station is removed, so the total number of key points drops from 49 to 48.  This is also handled automatically through the `downloadOFmodel.py` step above.
+
+
+<!--
+```yaml
+  FSTFile:
+    DT: 0.0008608815426997245 # Typically 0.005, make smaller for beamdyn
+    DT_Out: 0.0215220385675
+    CompElast:  2  # use beamdyn
+    CompInflow: 2
+    OutFileFmt: 3
+    AirDens:    1.000
+  EDFile:
+    NacYaw:   30.0
+    YawDOF:   False
+  AeroFile:
+    Wake_Mod: 0
+```
+-->
+
+### ALM settings
+
+When using the actuator line method in AMR-Wind, several additional parameters need to be set for this case.  The number of actuator line points along the length of the blade and tower needs to be set, as well as the magnitude of epislon, which sets the amount of smearing required for the applied force at each actuator force point.  These settings are defined in through these parameters in the AMR-Wind input file:
```

Comment:
"epislon" => "epsilon"

Reply:
"defined in through these"
Probably wanted just "through", but could also use "in" or "using"

Reply:
Fixed the typo.